docs/content/integrations/dbt/reference.mdx

```diff
@@ -763,57 +763,37 @@ Once your dbt models have been materialized, you can view the row count data in
 
 <Note>
   <strong>
-    Emitting column-level metadata as materialization metadata is currently an
-    experimental feature.{" "}
+    Emitting column-level metadata for dbt models is currently an experimental
+    feature.
   </strong>{" "}
-  To use this feature, you'll need to be on at least <code>dagster>=1.7.0</code>{" "}
-  and
-  <code>dagster-dbt>=0.23.0</code>.
+  To use this feature, you'll need to be on at least{" "}
+  <code>dagster>=0.17.9</code> and
+  <code>dagster-dbt>=0.23.9</code>.
```

Comment:
We should wait to land this for `1.8` so this constraint will be easier to read.

```suggestion
  <code>dagster>=1.8.0</code> and
  <code>dagster-dbt>=0.24.0</code>.
```